src/main/java/core/basesyntax/exception/ValidationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class ValidationException extends RuntimeException {

```

Comment:
For the future, it is better to give more specific name
```suggestion
public class RegistrationException extends RuntimeException {
```